pkg/storage/stores/indexshipper/compactor/util/index.go

```diff
@@ -1,14 +1,13 @@
-package compactor
+package util
```

Comment:
There was already code that converted filenames to chunk refs but it was in a package that couldn't be imported in the compactor without creating a circular dependency.

I moved it here and pointed all usages at it to break the circular dependency.